src/tool_writeout.c

```diff
@@ -127,6 +127,8 @@ static const struct writeoutvar variables[] = {
   {"time_starttransfer", VAR_STARTTRANSFER_TIME, CURLINFO_STARTTRANSFER_TIME_T,
    writeTime},
   {"time_total", VAR_TOTAL_TIME, CURLINFO_TOTAL_TIME_T, writeTime},
+  {"tls_earlydata", VAR_TLS_EARLYDATA_SENT, CURLINFO_EARLYDATA_SENT_T,
+   writeLong},
```

Comment:
```suggestion
   writeOffset},
```

Reply:
Yeah, this @jay discovery help explain a Windows problem.

Reply:
Thanks! Just pushed the change.